pycontw/pycontw-frontend
# Summary: rendered sponsor cards

## Types of changes


* **New feature**

## Description


need these components to display sponsorship

* my:
<img width="1512" alt="Screen Shot 2022-04-09 at 9 27 55 PM" src="https://user-images.githubusercontent.com/9366404/162576133-48f5f90a-3e12-471b-b2a5-ffc660fa73ab.png">

* figma: 👇  https://www.figma.com/file/W3hdepQ7v7ZGYRP6QctBXH/PyCon-Taiwan-%E5%AE%98%E7%B6%B2%E8%A6%96%E8%A6%BA-2022?node-id=0%3A1
<img width="1126" alt="Screen Shot 2022-04-09 at 9 25 25 PM" src="https://user-images.githubusercontent.com/9366404/162576195-80e4b199-2dac-4d44-96e3-a21e7a628d33.png">

## Steps to Test This Pull Request

1. `npm run dev`
2. navigate to <http://localhost:3000/2021/zh-hant>
3. scroll down and u'll see these sponsorship cards

## Expected behavior


* figma: 👇  https://www.figma.com/file/W3hdepQ7v7ZGYRP6QctBXH/PyCon-Taiwan-%E5%AE%98%E7%B6%B2%E8%A6%96%E8%A6%BA-2022?node-id=0%3A1
<img width="1126" alt="Screen Shot 2022-04-09 at 9 25 25 PM" src="https://user-images.githubusercontent.com/9366404/162576195-80e4b199-2dac-4d44-96e3-a21e7a628d33.png">

## Related Issue


N/A

## Additional context
